godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  TractViewController.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 4/24/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class TractViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        displayNavigationButtons()
+        displayTitle()
+    }
+    
+    fileprivate func displayNavigationButtons() {
+        navigationItem.rightBarButtonItem = createShareButton()
+        navigationItem.leftBarButtonItem = createHomeButton()
+    }
+
+    fileprivate func displayTitle() {
+        if parallelLanguageIsAvailable() {
+            navigationItem.titleView = languageSegmentedControl()
+        } else {
+            navigationItem.title = currentTractTitle()
+        }
+    }
+    
+    fileprivate func createHomeButton() -> UIBarButtonItem {
```

Comment:
Can you move this methods to BaseViewController? Please see how the buttons are created and loaded there, that will help us to reuse the buttons across viewcontrollers